### PR TITLE
DOC: examples/README.rst: link to binder-examples, binder-ready

### PR DESCRIPTION
**Pull request by westurner**

### CODE REVIEW

**Comment by betatim** [comment]
Yes?

**Comment by westurner** [comment]
#binderready would be more hashtaggable than binder-ready

**Comment by betatim** [comment]
For posterity/people from the future digging in the history could you edit the initial comment of this PR to explain briefly what the motivation for the change you are proposing is?

**Comment by westurner** [comment]
When I look at the examples directory, I wonder where examples that work
with binder/repo2docker are; so this PR adds a link to binder-examples.
Even though the directory is for examples of binder implementations, AFAIU

File: examples/README.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+See https://github.com/binder-examples for examples of https://github.com/topics/binder-ready Jupyter notebooks.
```
Comment (willingc) [inline examples/README.rst]: Let's rephrase:
```
See the [binder-examples](https://github.com/binder-examples) GitHub organization for examples of creating a binder-ready Jupyter project. For GitHub examples of third-party repositories using Binder, you may search GitHub for the [binder-ready](https://github.com/topics/binder-ready] and [binder](https://github.com/topics/binder) topics.
```

**Comment by willingc** [comment]
> When I look at the examples directory, I wonder where examples that work
with binder/repo2docker are; so this PR adds a link to binder-examples.
Even though the directory is for examples of binder implementations, AFAIU

See my suggested rephrasing on the PR. To clarify, binder-examples are examples of how to prepare your repo for use with repo2docker or mybinder.org. 

I'm not clear what you are referring to when you say "examples directory".

**Comment by westurner** [comment]
The modified README.rst is located in the examples/ directory of the repository.

**Pull request closed without merging**